packages/composites/multi-address-list/README.md

```diff
@@ -0,0 +1,101 @@
+|    ✅     |      coinbasePrime      |           |                                                    Input parameters for the Coinbase Prime EA                                                     |  object  |                                                 |           |            |                |
+|    ✅     | coinbasePrime.portfolio |           |                                                     The portfolio ID to query the balance of                                                      |  string  |                                                 |           |            |                |
+|    ✅     |  coinbasePrime.symbols  |           |                                                       The symbol to return the balance for                                                        | string[] |                                                 |           |            |                |
+|           |   coinbasePrime.type    |           |                                                            The balance type to return                                                             |  string  | `trading`, `vault`, `wallet_type_other`, `web3` |  `vault`  |            |                |
+|           |  coinbasePrime.chainId  |           |                                                           The ID of the chain to return                                                           |  string  |              `mainnet`, `testnet`               | `mainnet` |            |                |
+|           |  coinbasePrime.network  |           |                                                               The network to return                                                               |  string  |                                                 | `bitcoin` |            |                |
+|           | coinbasePrime.batchSize |           |                                                    The number of addresses to fetch at a time                                                     |  number  |                                                 |   `100`   |            |                |
+|           |  coinbasePrime.apiKey   |           | Alternative api keys to use for this request, {$apiKey}_ACCESS_KEY {$apiKey}\_PASSPHRASE {$apiKey}\_SIGNING_KEY required in environment variables |  string  |                                                 |           |            |                |
+|           | coinbasePrime.endpoint  |           |                                                        Endpoint name to make a request to                                                         |  string  |                                                 | `wallet`  |            |                |
+
+### Example
+
+Request:
+
+```json
+{
+  "data": {
+    "endpoint": "address-list",
+    "anchorage": {
```

Comment:
maybe silly question since i haven't looked at the code yet, but is the idea that for any given request, only the EAs listed in the request will be called?

Reply:
I changed this part. The EA will request only to sources that are in the request